catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -42,6 +42,20 @@ Implementation details
         <argument ref="queryResponseTransformers"/>
     </bean>
 
+    <reference-list id="listOfMessageValidators" interface="ddf.catalog.validation.MessageValidator"
+                    availability="optional">
+        <reference-listener bind-method="bindPlugin" unbind-method="unbindPlugin">
+            <bean class="org.codice.ddf.platform.util.SortedServiceList"/>
+        </reference-listener>
+    </reference-list>
+
+    <bean id="messageValidatorManager" class="org.codice.ddf.catalog.ui.query.cql.MessageValidatorManager">
+        <cm:managed-properties
+                persistent-id="org.codice.ddf.catalog.ui.query.cql.MessageValidatorManager"
+                update-strategy="container-managed"/>
+        <property name="validators" ref="listOfMessageValidators"/>
```

Comment:
✏️ If we used `<argument ref="listOfMessageValidators"/>` then we wouldn't need a setter in the class.

Reply:
I thought that the setter might make more sense, since `MessageValidator`s can appear or disappear asynchronously. If I get rid of the setter, wouldn't the application need to construct a new `MessageValidatorManager` after each change in the availability of a `MessageValidator`?